docs/source/servercli.rst

```diff
@@ -20,6 +20,7 @@ Fabric-CA Server's CLI
       -b, --boot string                               The user:pass for bootstrap admin which is required to build default config file
           --ca.certfile string                        PEM-encoded CA certificate file (default "ca-cert.pem")
           --ca.chainfile string                       PEM-encoded CA chain file (default "ca-chain.pem")
+          --ca.ignorecertexpiry                       Ignore Certificate Expirty for re-enroll
```

Comment:
Since it only applies to reenroll and not other functions, we should probably include that in the property name (at the risk of it being too long), e.g. `ignorereenrollcertexpiry`